core/src/main/java/com/hcl/ecomm/core/services/impl/CartServiceImpl.java

```diff
@@ -79,4 +84,54 @@ public int getCartItemCount(String cartId) {
        cartItemCount = cartItemsArray.size();
        return cartItemCount;
     }
+
+    @Override
+    public String updateCartDetails(String payload) {
```

Comment:
@DeepaliAgrwl : why is updateCartDetails method required ? you can reuse the same servlet Nithesh has created for cart item count

Reply:
@rishabh-behl  Nithesh is doing getcart and delete cart only. Updation is managed on locally on javascript. That is why I required this method